lib/dependencies.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Define the required Nextflow plugins.
+ */
+export const NEXTFLOW_PLUGINS: string[] = [
+  "nf-amazon@2.9.2",
+  "nf-schema@2.3.0",
+  "nf-wave@1.7.4",
```

Comment:
I got these from inspecting the container when running a test stub in dev. Instead of `nf-schema`, oncoanalyser 1.0.0 uses `nf-validation`. I'm not sure if we want to add this as well, or if `nf-schema` is enough.

Reply:
I'd only include what is directly needed by the built pipeline Docker image. This will likely need adjustment after oncoanalyser 2.0.0 has been released, which is the version will target with the CDK application for the PMCC panel use case